packages/functional-tests/tests/key-stretching-v2/signinBlocked.spec.ts

```diff
@@ -97,6 +97,6 @@ async function removeAccount(
   await settings.deleteAccountButton.click();
   await deleteAccount.deleteAccount(password);
 
-  await expect(page).toHaveURL(`${target.baseUrl}?delete_account_success=true`);
+  await expect(page).toHaveURL(`${target.baseUrl}`);
```

Comment:
I removed this because React doesn't need to pass this as a query param, and we still have the success message below it. This should pass with React or Backbone.